api/src/main/java/io/strimzi/api/kafka/model/CustomResourceConditions.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.api.kafka.model;
+
+import io.fabric8.kubernetes.client.CustomResource;
+import io.strimzi.api.kafka.model.status.Status;
+
+import java.util.function.Predicate;
+
+public class CustomResourceConditions {
+
+    public static <Y extends Status, T extends CustomResource<?, Y>> Predicate<T> isReady() {
+        return isStatusLatestGenerationAndMatches(anyCondition("Ready", "True"));
+    }
+
+
+    public static Predicate<KafkaRebalance> isProposalReady() {
```

Comment:
I wonder if we should give it a more clear name ... `isKafkaRebalanceProposalReady` => to make it more clear what it does, since ProposalReady applies only there.